packaging/win32/freeze_parsec.py

```diff
@@ -57,12 +57,12 @@ def main(parsec_source):
     if not TOOLS_VENV_DIR.is_dir():
         print(f"### Create tool virtualenv ###")
         run(f"python -m venv {TOOLS_VENV_DIR}")
-        run(f"{ TOOLS_VENV_DIR / 'Scripts/pip' } install wheel")
+        run(f"{ TOOLS_VENV_DIR / 'Scripts/python' } -m pip install wheel")
```

Comment:
Any reason to do so ?

Reply:
My environment had no pip shortcut for some reason, and I guess it's more robust this way.

Reply:
👍